Extensions/EdgeScrollCamera.json

```diff
@@ -0,0 +1,1473 @@
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Trigger width (BOTTOM edge)",
+          "longDescription": "",
+          "name": "TriggerWidthBottom",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Trigger width (LEFT edge)",
+          "longDescription": "",
+          "name": "TriggerWidthLeft",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Trigger width (RIGHT edge)",
+          "longDescription": "",
+          "name": "TriggerWidthRight",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Scroll speed (UP)",
+          "longDescription": "",
+          "name": "ScrollSpeedUp",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Scroll speed (DOWN)",
+          "longDescription": "",
+          "name": "ScrollSpeedDown",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Scroll speed (LEFT)",
+          "longDescription": "",
+          "name": "ScrollSpeedLeft",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Scroll speed (RIGHT)",
+          "longDescription": "",
+          "name": "ScrollSpeedRight",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Layer",
+          "longDescription": "",
+          "name": "Layer",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "layer"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Camera",
+          "longDescription": "",
+          "name": "Camera",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Check if the camera is scrolling.",
+      "fullName": "Is camera scrolling?",
```

Comment:
```suggestion
      "fullName": "Is camera scrolling",
```

Reply:
Should we also change the placeholder text on the extension editor?  (because it uses a question mark in the suggested text)